test/e2e/framework/gpu/gpu_util.go

```diff
@@ -24,5 +24,5 @@ const (
 	// GPUDevicePluginDSYAML is the official Google Device Plugin Daemonset NVIDIA GPU manifest for GKE
 	// TODO: Parametrize it by making it a feature in TestFramework.
 	// so we can override the daemonset in other setups (non COS).
-	GPUDevicePluginDSYAML = "https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml"
+	GPUDevicePluginDSYAML = "https://raw.githubusercontent.com/kubernetes/kubernetes/controlplane/cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml"
```

Comment:
We can't replace this master reference here because the path you're updating here references an existing file path. Switching it to control plane will point `GPUDevicePlugimDSYAML` to something that likely doesn't exist.

Might be best to leave this reference here so you don't have to worry about breaking things that depend on that path.

Reply:
@MorrisLaw I understand